docs/about-lightning.md

```diff
@@ -111,7 +111,8 @@ You have 3 options for exploring OpenFn/Lightning:
 2. To get your own account and start building non-production workflows, register
    for an account at [app.openfn.org](https://app.openfn.org/).
 3. To install and run Lightning locally follow the instructions in the
-   [github README](https://github.com/OpenFn/Lightning).
+   [github README](https://github.com/OpenFn/Lightning) or the 
+   [linked Docs site](https://github.com/OpenFn/lightning/). 
```

Comment:
same link. should the second be https://openfn.github.io/Lightning/readme.html ?